dietpi/dietpi-software

```diff
@@ -4166,6 +4173,55 @@ WantedBy=multi-user.target
 _EOF_
 		fi
 
+		if To_Install 141 adsb-feeder # ADSB Feeder
+		then
+			# create the userdata directory and clone the adsb-feeder repo into /tmp
+			G_EXEC mkdir -p /mnt/dietpi_userdata/adsb-feeder
+			G_EXEC_OUTPUT=1 G_EXEC git clone -b dietpi 'https://github.com/dirkhh/adsb-feeder-image.git' /tmp/adsb-feeder
+
+			# remove the service that isn't needed for an app install on DietPi and install the rest
+			G_EXEC cd /tmp/adsb-feeder/src/modules/adsb-feeder/filesystem/root
+			G_EXEC rm -rf ./usr/lib/systemd/system/adsb-bootstrap.service
+			G_EXEC cp -a ./usr/lib/systemd/system/* /etc/systemd/system/
+
+			# determine the version
+			ADSB_FEEDER_DATE_COMPONENT=$(git log -20 --date=format:%y%m%d --format="%ad" | uniq -c | head -1 | awk '{ print $2"."$1 }')
+			ADSB_FEEDER_TAG_COMPONENT=$(git describe --match "v[0-9]*" | cut -d- -f1)
+			ADSB_FEEDER_VERSION="${ADSB_FEEDER_TAG_COMPONENT}(dietpi)-${ADSB_FEEDER_DATE_COMPONENT}"
+
+			# create the target directory for the app and populated with the code from the git checkout
+			G_EXEC mkdir -p /opt
+			G_EXEC cp -a /tmp/adsb-feeder/src/modules/adsb-feeder/filesystem/root/opt/adsb /opt
```

Comment:
`/opt` is pre-generated as failsafe step on every install run. But if that directory already exists, I think we should remove it, i.e. doing a clean reinstall? As you have an update service, a reinstall only makes sense when it is really broken, and then it makes sense to make it as clean as possible, excluding only data and configs.
```suggestion
			[[ -d '/opt/adsb' ]] && G_EXEC rm -R /opt/adsb
			G_EXEC cp -a /tmp/adsb-feeder/src/modules/adsb-feeder/filesystem/root/opt/adsb /opt
```
As I see it, can we use `/opt/adsb-feeder` and `/opt/adsb-feeder-update`, to have them unique and a little aligned? I see the letter is generated by `adsb-feeder-update.service`, so it is okay to leave it as it is if this causes you too much hassle in your code.

Reply:
I can make that change... that requires making changes both in my adsb-feeder-image upstream repo and here, but I can do that today. I should have plenty of time to wrestle with that (famous last words)

Reply:
actually... somewhat. `/opt/adsb` is really well established as the folder to use for this setup. Almost all of the documentation that enthusiasts might use will point to that. I would not be thrilled to have to rename that one.
But I did rename `/opt/feeder-update` to `/opt/adsb-feeder-update`
So now they both start with `/opt/adsb`

Reply:
Okay should be sufficiently unique.